test/eeprom.py

```diff
@@ -0,0 +1,114 @@
+import tkinter as tk
+from datetime import date, datetime
+from tkinter import Button, Entry, Label
```

Comment:
Will need a separate requirements.txt file (maybe requirements_prod_test.txt). TK doesn't always ship with python

Reply:
I created a new commit to add that